packages/manager/modules/iam/src/components/policyIdentities/policyIdentities.controller.js

```diff
@@ -0,0 +1,106 @@
+import { ROUTES } from '@iam/routes';
+import { decodeUrn, encodeUrn } from '@iam/resolves';
+import { URN_VERSION, ENTITY } from '@iam/constants';
+
+export default class PolicyIdentitiesController {
+  /* @ngInject */
+  constructor($q, $translate, IdentityService, PolicyService, coreConfig) {
+    this.$q = $q;
+    this.$translate = $translate;
+    this.IdentityService = IdentityService;
+    this.PolicyService = PolicyService;
+    this.region = coreConfig.getRegion();
+    this.user = coreConfig.getUser();
+  }
+
+  $onInit() {
+    this.loading = true;
+
+    this.identities = this.policy.identities
+      .map(decodeUrn)
+      .map((urn) => ({ title: urn.components.slice(-1).toString(), urn }));
+
+    this.$q
+      .all({
+        users: this.IdentityService.getUsers(),
+        groups: this.IdentityService.getGroups(),
+      })
+      .then(({ users, groups }) => {
+        this.userList = [
+          ...users.map((user) => ({ accountType: 'user', userName: user })),
+          ...groups.map((group) => ({
+            accountType: 'group',
+            userName: group,
+          })),
+        ];
+      })
+      .catch(() => {
+        this.userList = [];
+      })
+      .finally(() => {
+        this.loading = false;
+      });
+  }
+
+  goToDeleteIdentity(identity) {
+    return this.goTo({
+      name: ROUTES.DELETE_IDENTITY,
+      params: { identity },
+    });
+  }
+
+  keypressTrackEvent(key) {
+    if (key !== 'Enter') {
+      this.suggestion = null;
+    }
+  }
```

Comment:
Keep consistent names with the input
Pass the event instead of the key
```suggestion
  onUsersFieldKeypress(event) {
    if (event.key !== 'Enter') {
      this.suggestion = null;
    }
  }
```

Reply:
It's applied